views/dashboard/signatories.ejs

```diff
@@ -1,6 +1,23 @@
 <section class="signatories" id="signatures">
-    <h5>Signed,</h5>
-
+    <div class="signatory-header">
+        <h5>Signed,</h5>
+        <div class="signatory-sort border rounded px-1">
+            <form name="signatory-sort" method="post" action="/save-sort">

```

Comment:
I'd change the method go `GET`, might be useful to store the sort order in the URL actually